udata_tabular_preview/views.py

```diff
@@ -9,12 +13,23 @@
                       static_folder='static')
 
 
-@blueprint.route('/preview/')
-def preview():
-    if current_app.config.get('PREVIEW_MODE') is None:
-        abort(404)
-    return render_template('tabular/preview.html')
+def can_explore_dataset(ctx):
+    dataset = ctx.get('dataset', None)
+    return dataset and any(can_explore(resource) for resource in dataset.resources)
+
+
+@template_hook('header.snippets', when=can_explore_dataset)
+def load_explore_script(ctx):
+    dataset = ctx.get('dataset', None)
+    resources = []
+    for resource in dataset.resources:
+        if can_explore(resource):
+            resources.append(resource.id)
+    return theme.render('metadata.html', resources=resources)
```

Comment:
See previous comment: I'm not sure we need this anymore?

Reply:
Already discuss on udata-front : https://github.com/etalab/udata-front/pull/169#discussion_r1039361195
We'll keep this for now